src/repl.zig

```diff
@@ -110,311 +112,104 @@ pub fn ReplType(comptime MessageBus: type) type {
             }
         }
 
-        const prompt = "> ";
+        fn redraw_line(repl: *const Repl, prompt_str: []const u8) !void {
+            const buf = repl.line_editor.get_buffer();
+            const prompt_len = prompt_str.len;
+
+            // move to start of line (CR)
```

Comment:
Comments [should be full sentences](https://github.com/tigerbeetle/tigerbeetle/blob/main/docs/TIGER_STYLE.md#naming-things).
```suggestion
            // Move to the start of the line (CR).
```